<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundTaskProgress</class>
 <widget class="QWidget" name="BackgroundTaskProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblTaskName">
     <property name="text">
      <string>Phantom Task</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2" alignment="Qt::AlignBottom">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>X</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QProgressBar" name="pbProgress">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
